tests/experimental/datasources/tasks.py

```diff
@@ -0,0 +1,36 @@
+"""
+CD to this directory to run these invoke commands.
+"""
+
+from __future__ import annotations
+
+import pathlib
+
+import invoke
+import pandas as pd
+
+TEST_ROOT = pathlib.Path(__file__).parent.parent.parent.resolve(strict=True)
+
+
+@invoke.task(aliases=["gen-assets"])
+def generate_asset_files(
+    ctx: invoke.Context,
+    source_csv_dir: str | pathlib.Path = None,
+    limit: int = 12,
+    year: str = 2019,
+):
+    """Generate or re-generate various test asset files."""
```

Comment:
This is how I generated all the json files in json assets.
We really shouldn't need to run it again unless we want tweak it to generate other pandas formats.

The other thing we could do is just generate these other formats on the fly with some parametrized fixture.
But I didn't want to spend more time going down that rabbit hole.
We discussed before that there's limited value testing all of these various formats because then we are essentially just testing that `pandas` works. 😆